### DISCUSSION TITLE
Возможность настройки зеркала

### DISCUSSION THREAD

**Discussion by eGreyWolf** [quarckster/kodi.kino.pub#212] [Ideas]
Уважаемый Дмитрий!
В очередной раз СПАСИБО Вам заглавными буквами.

Далее, вопрос-идея-пожелание: 
1. Нет ли смысла добавить в настройки плагина возможность оговаривать адрес персонального "зеркала", полученного при активации профиля PRO, от mos-gorsud.net? 

2. Нет ли смысла сделать, пусть так же, в настройках плагина, возможность переключения между серверами Kino.Pub-а? Их, вроде как, всего два. Но маршруты до них - разные. Равно, как и количество хопов, как и наличие дропов. Починить "сетевые" проблемы ни кто не может, даже оператор. Спасает "переключение" между площадками. А с Kodi - и переключения, увы, как возможности, нет.

3. Не идея, а старый вопрос: почему-то (не смотря на актуальную версию Вашего плагина) нет возможности хотя бы читать "Отзывы". Хотя в описании к нему, у Вас, есть ремарка:
_added custom context menu items: "Комментарии kino.pub"_

Еще раз СПАСИБО!
И удачи!

**Comment by stavultras** [comment]
>     1. Нет ли смысла добавить в настройки плагина возможность оговаривать адрес персонального "зеркала", полученного при активации профиля PRO, от mos-gorsud.net?
Плагин работает не через browser url, а через API url, который у кинопаба всегда один - на сколько я знаю.
Зеркала это для браузеров только. Если вы именно это имели ввиду.
>     3. Не идея, а старый вопрос: почему-то (не смотря на актуальную версию Вашего плагина) нет возможности хотя бы читать "Отзывы". Хотя в описании к нему, у Вас, есть ремарка:
>         _added custom context menu items: "Комментарии kino.pub"_

С этим то как раз таки никаких проблем: Свойства элемента ->
![](https://clip2net.com/clip/m378127/5d05c-clip-68kb.jpg)
![](https://clip2net.com/clip/m378127/ff201-clip-122kb.png)

**Reply by eGreyWolf** [reply]
Доброго времени, Дмитрий!
Хорошо, спасибо.
На сколько я понимаю логику КП, то зеркала не только "для браузера": у меня "моё" зеркало прописано в настройках КиноПаба на AppleTV в двух местах: это сам профиль и злосчастный сертификат. 
Иными словами, AppleTV ходит за контентом не на kino.pub, а на его "зеркало". 

И последний, пожалуй, нюанс: в ЛК КиноПаба можно просмотреть КУДА стучится какой клиент. Коди - всегда в Нидерланды. AppleTV - в Россию. От чего и возникла идея предложить Вам реализовать возможность ПЕРЕключения меж серверами. А в свете того, что сервер в России более производительный, то идея казалась привлекательной.

В качестве еще одной идеи, Дмитрий, с Вашего позволения: нельзя реализовать нечто подобное в браузерной версии ФИЛЬТР. Там удобно искать, выставить в нем критерии отбора. Например, не ниже рейтинга IMDB 6.8.

Спасибо!

**Reply by quarckster** [reply]
> На сколько я понимаю логику КП, то зеркала не только "для браузера": у меня "моё" зеркало прописано в настройках КиноПаба на AppleTV в двух местах: это сам профиль и злосчастный сертификат.
Иными словами, AppleTV ходит за контентом не на kino.pub, а на его "зеркало".

насчёт этого не уверен, надо уточнить у команды КП.

> В качестве еще одной идеи, Дмитрий, с Вашего позволения: нельзя реализовать нечто подобное в браузерной версии ФИЛЬТР. Там удобно искать, выставить в нем критерии отбора. Например, не ниже рейтинга IMDB 6.8.

Это проблематично сделать через UI Kodi. Он просто не предоставляет возможности создать форму с полями.

**Comment by quarckster** [comment]
> Нет ли смысла добавить в настройки плагина возможность оговаривать адрес персонального "зеркала", полученного при активации профиля PRO, от mos-gorsud.net?

Насколько я понимаю, зеркало для работы плагина не требуется, оно необходимо для входа на сайт через бразуер.

> Нет ли смысла сделать, пусть так же, в настройках плагина, возможность переключения между серверами Kino.Pub-а? Их, вроде как, всего два. Но маршруты до них - разные. Равно, как и количество хопов, как и наличие дропов. Починить "сетевые" проблемы ни кто не может, даже оператор. Спасает "переключение" между площадками. А с Kodi - и переключения, увы, как возможности, нет.

Да, это имеет смысл и API предоставляет такую возможность.

> Не идея, а старый вопрос: почему-то (не смотря на актуальную версию Вашего плагина) нет возможности хотя бы читать "Отзывы". Хотя в описании к нему, у Вас, есть ремарка:
added custom context menu items: "Комментарии kino.pub"

Ответили выше

**Comment by eGreyWolf** [comment]
Спасибо Вам, в очередной раз, за Ваш, такой заметный и важный проект, труд!

ср, 10 нояб. 2021 г. в 11:48, Dmitrii Misharov ***@***.***>:

> На сколько я понимаю логику КП, то зеркала не только "для браузера": у
> меня "моё" зеркало прописано в настройках КиноПаба на AppleTV в двух
> местах: это сам профиль и злосчастный сертификат.
> Иными словами, AppleTV ходит за контентом не на kino.pub, а на его
> "зеркало".
>
> насчёт этого не уверен, надо уточнить у команды КП.
>
> В качестве еще одной идеи, Дмитрий, с Вашего позволения: нельзя
> реализовать нечто подобное в браузерной версии ФИЛЬТР. Там удобно искать,
> выставить в нем критерии отбора. Например, не ниже рейтинга IMDB 6.8.
>
> Это проблематично сделать через UI Kodi. Он просто не предоставляет
> возможности создать форму с полями.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quarckster/kodi.kino.pub/discussions/212#discussioncomment-1617771>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASG62TQZPG33PJYQ7NQJFN3ULIWVDANCNFSM5HWCCGDQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Whith great regards,
---
GW

**Comment by quarckster** [comment]
В версии 3.1.0 я добавил настройку локации CDN.

**Comment by eGreyWolf** [comment]
Вы, Дмитрий, небывало КРУТОЙ!
СПАСИБО!

пт, 19 нояб. 2021 г. в 16:54, Dmitrii Misharov ***@***.***>:

> В версии 3.1.0 я добавил настройку локации CDN.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quarckster/kodi.kino.pub/discussions/212#discussioncomment-1670656>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASG62TWXDER7FGUYFV7AZRDUMZJJDANCNFSM5HWCCGDQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Whith great regards,
---
GW

**Comment by eGreyWolf** [comment]
Пользуясь Вашей добротой еще спрошу/скажу.
Смотрю (или стараюсь смотреть, если есть такая возможность) со звуком AC3
<https://ru.wikipedia.org/wiki/Dolby_Digital>. По какой-то иронии, среди
всех прочих звуковых дорожек, в интерфейсе плагина настроек, выбора
дорожки, AC3-дорожка всегда находится в самом внизу. Вы, как автор,
предполагали, что для выбора звуковой дорожки, пользователь кликает в
иконку. Однако, в случае с "самой нижней дорожкой" это сложно по нескольким
причинам. Приходится раскрывать "шестеренку" настроек и глазами выбирать
AC3 (в моем случае, порой, "одну из"). Вопрос: нельзя ли в настройках, раз
и навсегда, закрепить приоритет выбора на тип AC3?
Связанный со звуком, косвенный вопрос: нельзя ли добавить фильтр (по
аналогии с отфильтровкой, точнее исключением аниме или рекламы), отбирать
фильмы лишь с AC3?
Спасибо.

пт, 19 нояб. 2021 г. в 16:55, Grey Wolf ***@***.***>:

> Вы, Дмитрий, небывало КРУТОЙ!
> СПАСИБО!
>
> пт, 19 нояб. 2021 г. в 16:54, Dmitrii Misharov ***@***.***>:
>
>> В версии 3.1.0 я добавил настройку локации CDN.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/quarckster/kodi.kino.pub/discussions/212#discussioncomment-1670656>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASG62TWXDER7FGUYFV7AZRDUMZJJDANCNFSM5HWCCGDQ>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>
>
> --
> Whith great regards,
> ---
> GW
>


-- 
Whith great regards,
---
GW

**Reply by quarckster** [reply]
> Вопрос: нельзя ли в настройках, раз и навсегда, закрепить приоритет выбора на тип AC3?

Нет, к сожалению, это сделать нельзя. Kodi не предоставляет API для этого.

> нельзя ли добавить фильтр (по аналогии с отфильтровкой, точнее исключением аниме или рекламы), отбирать фильмы лишь с AC3?

Это выполнимо, но мне эта функция не нужна. Буду рад патчам.

**Comment by eGreyWolf** [comment]
Спасибо! Это просто не критические хотелки.
Что касается выбора типа "озвучки", то хотелка была из разряда "юзабилити",
ибо, реально, с KODI переключаться между дорожками не удобно.
Например, в том же "Foundation" звуковых дорожек с десяток и AC3, по
какой-то нелепой логике - в самом низу.
Спасибо еще раз, за Ваш труд!

пт, 19 нояб. 2021 г. в 17:13, Dmitrii Misharov ***@***.***>:

> Вопрос: нельзя ли в настройках, раз и навсегда, закрепить приоритет выбора
> на тип AC3?
>
> Нет, к сожалению, это сделать нельзя. Kodi не предоставляет API для этого.
>
> нельзя ли добавить фильтр (по аналогии с отфильтровкой, точнее исключением
> аниме или рекламы), отбирать фильмы лишь с AC3?
>
> Это выполнимо, но мне эта функция не нужна. Буду рад патчам.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quarckster/kodi.kino.pub/discussions/212#discussioncomment-1670765>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASG62TUCXMPQRDXJDWUVI6LUMZLQ5ANCNFSM5HWCCGDQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Whith great regards,
---
GW
